README.md

```diff
@@ -57,13 +57,17 @@ Commands:
 You can use caching by including this module in your ActiveRecord models,
 
     class OfferEvent < ActiveRecord::Base
-      include Gris::CacheKey
+      include Gris::Caching
     end
 
 and then cache inside GET requests like this
 
     offer_event =  OfferEvent.cached_find(id)
 
+To expire the cache for an object, call
+
+    OfferEvent.expire_cache(id)
```

Comment:
Yep. It's not a huge deal but in cases where we already have the instance I think it may be more convenient than dropping to the class method. 

I don't feel too strongly but take the following or leave it. 

```ruby
module Gris
  module Caching
    extend ActiveSupport::Concern

    included do
      def expire_cache
        self.class.expire_cache_for(id)
      end
    end

    class_methods do
      def cached_find(id)
        Gris.cache.fetch(cache_key_for(id)) do
          find(id)
        end
      end

      def expire_cache_for(id)
        Gris.cache.delete(cache_key_for(id))
      end

      private

      def cache_key_for(id)
        "#{model_name.cache_key}/#{id}"
      end
    end
  end
end
```

```ruby
require 'spec_helper'

describe 'Gris::Caching' do
  let(:id) { SecureRandom.random_number(500) }

  it 'uses the cache for cached_find' do
    foo = CachingHelper.cached_find(id)
    expect(foo.id).to eq(id)
    allow(Gris.cache).to receive(:fetch).and_return('cached-result')
    expect(CachingHelper.cached_find(id)).to eq('cached-result')
  end

  it 'expires cache for a given id' do
    allow(Gris.cache).to receive(:delete).with("cache_key_helper/#{id}")
    CachingHelper.expire_cache_for(id)
    expect(Gris.cache).to have_received(:delete).with("cache_key_helper/#{id}")
  end

  it 'expires cache for a given instance' do
    allow(Gris.cache).to receive(:delete).with("cache_key_helper/#{id}")
    instance = CachingHelper.new(id: id)
    instance.expire_cache
    expect(Gris.cache).to have_received(:delete).with("cache_key_helper/#{id}")
  end
end
```